README.md

```diff
@@ -148,6 +148,8 @@ setting with the following possible values: `EventsTransportSSE` and `EventsTran
 See [Event Stream](https://mesosphere.github.io/marathon/docs/rest-api.html#event-stream) and
 [Event Subscriptions](https://mesosphere.github.io/marathon/docs/rest-api.html#event-subscriptions) for details.
 
+Event subscriptions can also be individually controlled with the `Subscribe` and `Unsubscribe` functions.
```

Comment:
Could you refer to the `Controlling subscriptions` section that you added below from here?